back/engines/commercial/verification/app/services/verification/verification_service.rb

```diff
@@ -112,17 +112,19 @@ def make_verification(user:, method_name:, uid:)
         end
       end
 
-      verification = ::Verification::Verification.new(
+      verification = ::Verification::Verification.find_or_initialize_by(
```

Comment:
Before, we created a new verification record even if it exists in the DB. I didn't find the reasons why we may need it.

Reply:
I think it's just as a kind of verification log, so we know when people verified, and how 'old' the verification is. That might be useful if we want to invalidate based on time, for example. Any issue with leaving it as is?

Reply:
No, not really. I just noticed that we usually do not create new SSO identities https://github.com/CitizenLabDotCo/citizenlab/blob/89b584ea2afbaf20af7ab4b2985a0660e6f8b72d/back/app/models/identity.rb#L33
But for Vienna, we did, and it was an indicator of a bug. And so, I decided to keep it the same in two cases.

But maybe indeed verifications usually do not happen twice, so we can keep this log.